graphql/directive_test.go

```diff
@@ -891,3 +891,62 @@ func TestRequirePatchOwner(t *testing.T) {
 		})
 	}
 }
+func TestRequireAdmin(t *testing.T) {
+	defer func() {
+		err := db.ClearCollections(user.Collection)
+		require.NoError(t, err, "unable to clear user collection")
+	}()
+
+	setupPermissions(t)
+
+	usr, err := setupUser(t)
+	require.NoError(t, err, "Error while setting up user")
+	require.NotNil(t, usr, "User setup returned nil")
+
+	ctx := context.Background()
```

Comment:
I think nowadays we recommend to use the testing context instead of creating a new context, i.e. `t.Context()`